lib/territory.js

```diff
@@ -19,7 +19,7 @@ export function purchasedType (sub) {
 export function proratedBillingCost (sub, newBillingType) {
   if (!sub ||
     sub.billingType === 'ONCE' ||
-    sub.billingType === newBillingType.toUpperCase()) return null
+    sub.billingType === newBillingType.toUpperCase()) return 0
```

Comment:
not sure why this returned `null` instead of 0 (historical reasons?), but it's only used on territory update and afaict, it's safe to return 0